lib/harUtil.js

```diff
@@ -101,4 +102,11 @@ harUtil.createHarEntry = ({proxyTarget, responseBody, req, res}) => {
     request,
     response
   };
-}
+};
+
+harUtil.loadHarFromFile = scenarioFileName => {
+  const scenarioName = path.basename(scenarioFileName, '.har');
+  const scenarioFileLoc = scenarioFileName; //path.join(app.tapeDir, scenarioFileName);
+  const scenario = fs.readJsonSync(scenarioFileLoc);
```

Comment:
we can get rid of line 109 and just rename `scenarioFileLoc` to `scenarioFileName`